### PR TITLE
Make a bunch of edits

### PR DESCRIPTION
**Pull request by kylefiedler**
- Add Coworking spaces images
- Add Sponsors
- Rearrange homepage
- Make logo
- Add logos for incubators

### CODE REVIEW

File: _sass/base/_variables.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Typography
 $sans-serif: "ff-meta-web-pro", $helvetica;
-$serif: "droid-serif", $georgia;
+$serif: "ff-meta-serif-web-pro", $georgia;
```
Comment (joshuaogle) [inline _sass/base/_variables.scss:3]: Good choice :+1:

File: _sass/layout/_logo.scss [hunk 2]
```diff
@@ -1,13 +1,28 @@
...
-    color: $base-accent-color;
-    font-family: $base-font-family;
-    font-size: 1.5em;
-    font-weight: bold;
-    margin-right: 0.25em;
+  a {
+    span {
+      color: white;
+
+      &:before {
```
Comment (joshuaogle) [inline _sass/layout/_logo.scss:8]: Should be `&::before`, if you care about that

File: _sass/layout/_logo.scss [hunk 3]
```diff
@@ -1,13 +1,28 @@
...
+        @include size(2px 30px);
+        background-color: rgba(white, 0.5);
+        content: "";
+        display: inline-block;
+        margin: 0 0.75em -8px 0.25em;
```
Comment (joshuaogle) [inline _sass/layout/_logo.scss]: Want to convert the px to em?

**Comment by joshuaogle** [comment]
Looks good aside from those small notes :+1:

File: _layouts/default.html [hunk 4]
```diff
@@ -82,7 +83,13 @@ <h1 class="logo">
...
         <section class="logo-wrapper">
-          <h1 class="logo"><abbr>PSL</abbr>Philly Startup Leaders</h1>
+          <h1 class="logo">
+            <a href="/">
+              <abbr class="short">PSL</abbr>
+              <span class="full">Philly Startup Leaders</span>
+            </a>
+          </h1>
+
           <p>This site and it’s code are open source and licenced under MIT license. <a href="https://github.com/thoughtbot/psl">View the source</a>.</p>
```
Comment (whmii) [inline _layouts/default.html]: "its" not "it's"

File: _sass/base/_variables.scss (hunk 1 above)
Comment (whmii) [inline _sass/base/_variables.scss:3]: +1

File: coworking/index.html [hunk 5]
```diff
@@ -36,6 +43,9 @@ <h3 class="location">Rittenhouse Square</h3>
         <h2>Indy Hall</h2>
         <h3 class="location">Old City</h3>
       </header>
+      <div class="image">
+        <img src="/images/coworking/indy-hall.jpg" alt="" />
```
Comment (whmii) [inline coworking/index.html]: It'd be nice to have alt info on these images

File: images/external.svg [hunk 6]
```diff
@@ -1,15 +1,15 @@
-<?xml version="1.0" encoding="iso-8859-1"?>
-<!-- Generator: Adobe Illustrator 15.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
-<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
-<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="12px"
-	 height="12px" viewBox="0 0 12 12" style="enable-background:new 0 0 12 12;" xml:space="preserve">
-<g id="Icons" style="opacity:0.75;">
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 18.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```
Comment (whmii) [inline images/external.svg]: should be

```
<svg version="1.1" xmlns="http://www.w3.org/2000/svg"  mlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="-299 391 12 12">
```

File: images/external.svg [hunk 7]
```diff
@@ -1,15 +1,15 @@
-<?xml version="1.0" encoding="iso-8859-1"?>
-<!-- Generator: Adobe Illustrator 15.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
-<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
-<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="12px"
-	 height="12px" viewBox="0 0 12 12" style="enable-background:new 0 0 12 12;" xml:space="preserve">
-<g id="Icons" style="opacity:0.75;">
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (whmii) [inline images/external.svg]: :scissors:

File: images/external.svg [hunk 8]
```diff
@@ -1,15 +1,15 @@
...
+<!-- Generator: Adobe Illustrator 18.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```
Comment (whmii) [inline images/external.svg]: :scissors:

File: images/external.svg [hunk 9]
```diff
@@ -1,15 +1,15 @@
...
-		<polygon id="arrow_13_" style="fill-rule:evenodd;clip-rule:evenodd;" points="6.211,2 10,2 10,5.789 8.579,4.368 6.447,6.5
-			5.5,5.553 7.632,3.421 		"/>
+		<polygon id="box" fill="#FFFFFF" points="-297,393 -294,393 -294,394 -296,394 -296,400 -290,400 -290,398 -289,398 -289,401 
+			-297,401 		"/>
+		<polygon id="arrow_13_" fill="#FFFFFF" points="-292.8,393 -289,393 -289,396.8 -290.4,395.4 -292.6,397.5 -293.5,396.6 
+			-291.4,394.4 		"/>
 	</g>
 </g>
-<g id="Guides" style="display:none;">
+<g id="Guides" display="none">
```
Comment (whmii) [inline images/external.svg]: ¿Por que?

File: sponsors/index.html [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+  <article class="featured">
+    <h2>Want to sponsor PSL or an upcoming event?</h2>
+    <a href="mailto:engage@phillystartupleaders.org" class="main-action">Shoot us an email</a>
+  </article>
+
+  <article class="current-sponsors">
+    <h1>Thank you to our wonderful sponsors for helping us keep the Philly startup community strong.</h1>
+    <ul>
+      <li>
+        <a href="">
```
Comment (whmii) [inline sponsors/index.html]: Are these supposed to go anywhere?

**Pull request closed without merging**